Cannot compile with `jwt_auth` Feature Disabled

**Description**

It looks like there's a few places where `jsonwebtoken` is used that are not guarded by the `auth_jwt` feature. This causes Loco to fail to compile with the feature disabled.
```
error[E0433]: failed to resolve: use of undeclared crate or module `jsonwebtoken`
  --> /Users/tommy.mccormick/.cargo/registry/src/index.crates.io-6f17d22bba15001f/loco-rs-0.13.2/src/model/mod.rs:30:17
   |
30 |     Jwt(#[from] jsonwebtoken::errors::Error),
   |                 ^^^^^^^^^^^^ use of undeclared crate or module `jsonwebtoken`
   |
help: consider importing this module
   |
6  + use crate::errors;
   |
help: if you import `errors`, refer to it directly
   |
30 -     Jwt(#[from] jsonwebtoken::errors::Error),
30 +     Jwt(#[from] errors::Error),
   |
```

**To Reproduce**

Please outline the steps to replicate the bug, and if possible, provide a comprehensive code example consisting of both `main.rs` and `Cargo.toml`` files. A complete and functional code snippet would be highly appreciated.

**Expected Behavior**

Able to build the project with the `auth_jwt` feature disabled.

**Environment:**

**Additional Context**

Cargo.toml
```
loco-rs = { version = "0.13.2", default-features = false, features = [
    "cli",
    "with-db",
    "cache_inmem",
] }
```

--- a/src/model/mod.rs
+++ b/src/model/mod.rs
@@ -26,6 +26,7 @@ pub enum ModelError {
     #[error("{errors:?}")]
     ModelValidation { errors: ModelValidation },
 
+    #[cfg(feature = "auth_jwt")]
     #[error("jwt error")]
     Jwt(#[from] jsonwebtoken::errors::Error),
 

